tsconfig.base.json

```diff
@@ -27,6 +27,7 @@
     "noImplicitOverride": true,
     "skipLibCheck": true,
     "paths": {
+      // "@paima/mqtt/*": ["./packages/mqtt/*"],
```

Comment:
This won't work.

Packages in Paima need to be at depth 2 from the root folder (ex: `/packages/.../mqtt`) otherwise it won't work

Reply:
Thanks!